spec/requests/submissions_search_spec.rb

```diff
@@ -479,6 +479,31 @@
       end
     end
 
+    context "with high_value query for NSM" do
+      before do
+        create(:submission, :with_nsm_version,
+               high_value: true,
+               laa_reference: "LAA-AAAAA1")
+
+        create(:submission, :with_nsm_version,
+               high_value: true,
+               laa_reference: "LAA-AAAAA2")
+
+        create(:submission, :with_nsm_version,
```

Comment:
Worth making this explicitly `high_value: false` so you can prove you're correctly casting to a boolean?